test/Land/Model/test_bc.jl

```diff
@@ -0,0 +1,167 @@
+# Test that the way we specify boundary conditions works as expected
+using MPI
+using OrderedCollections
+using StaticArrays
+using Statistics
+
+using CLIMAParameters
+struct EarthParameterSet <: AbstractEarthParameterSet end
+const param_set = EarthParameterSet()
+
+using ClimateMachine
+using ClimateMachine.Land
+using ClimateMachine.Land.SoilWaterParameterizations
+using ClimateMachine.Mesh.Topologies
+using ClimateMachine.Mesh.Grids
+using ClimateMachine.DGMethods
+using ClimateMachine.DGMethods.NumericalFluxes
+using ClimateMachine.DGMethods: BalanceLaw, LocalGeometry
+using ClimateMachine.MPIStateArrays
+using ClimateMachine.GenericCallbacks
+using ClimateMachine.ODESolvers
+using ClimateMachine.VariableTemplates
+using ClimateMachine.SingleStackUtils
+
+FT = Float64;
```

Comment:
Same, let's put this inside a @testset block

Reply:
oh yeah! forgot that one!